benchmarks/src/main/java/org/elasticsearch/benchmark/compute/operator/EvalBenchmark.java

```diff
@@ -255,19 +255,19 @@ private static EvalOperator.ExpressionEvaluator evaluator(String operation) {
     }
 
     private static FieldAttribute longField() {
-        return new FieldAttribute(Source.EMPTY, "long", new EsField("long", DataType.LONG, Map.of(), true));
+        return new FieldAttribute.FieldAttirbuteBuilder(Source.EMPTY, "long", new EsField("long", DataType.LONG, Map.of(), true)).build();
     }
 
     private static FieldAttribute doubleField() {
-        return new FieldAttribute(Source.EMPTY, "double", new EsField("double", DataType.DOUBLE, Map.of(), true));
+        return new FieldAttribute.FieldAttirbuteBuilder(Source.EMPTY, "double", new EsField("double", DataType.DOUBLE, Map.of(), true)).build();
     }
 
     private static FieldAttribute intField() {
-        return new FieldAttribute(Source.EMPTY, "int", new EsField("int", DataType.INTEGER, Map.of(), true));
+        return new FieldAttribute.FieldAttirbuteBuilder(Source.EMPTY, "int", new EsField("int", DataType.INTEGER, Map.of(), true)).build();
```

Comment:
Spelling.

Maybe should just be `FieldAttribute.builder()` or `new FieldAttribute.Builder()`?